2021-07-17 11:41:35,276 WARN  [org.keycloak.services.resources.admin.UserResource] (default task-20) Could not update user password.: org.keycloak.models.ModelException: invalidPasswordMinSpecialCharsMessage
	at org.keycloak.keycloak-services@12.0.4//org.keycloak.credential.PasswordCredentialProvider.createCredential(PasswordCredentialProvider.java:79)
	at org.keycloak.keycloak-services@12.0.4//org.keycloak.credential.PasswordCredentialProvider.updateCredential(PasswordCredentialProvider.java:213)
	at org.keycloak.keycloak-services@12.0.4//org.keycloak.credential.UserCredentialStoreManager.lambda$updateCredential$7(UserCredentialStoreManager.java:194)
	at java.base/java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.LinkedList$LLSpliterator.tryAdvance(LinkedList.java:1253)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.anyMatch(ReferencePipeline.java:528)
	at org.keycloak.keycloak-services@12.0.4//org.keycloak.credential.UserCredentialStoreManager.updateCredential(UserCredentialStoreManager.java:194)
	at org.keycloak.keycloak-services@12.0.4//org.keycloak.services.resources.admin.UserResource.resetPassword(UserResource.java:577)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:138)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:543)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:432)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$0(ResourceMethodInvoker.java:393)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.interception.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:358)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:395)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:364)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invokeOnTargetObject(ResourceLocatorInvoker.java:150)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invoke(ResourceLocatorInvoker.java:110)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invokeOnTargetObject(ResourceLocatorInvoker.java:141)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invoke(ResourceLocatorInvoker.java:110)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invokeOnTargetObject(ResourceLocatorInvoker.java:141)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invoke(ResourceLocatorInvoker.java:110)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invokeOnTargetObject(ResourceLocatorInvoker.java:141)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.ResourceLocatorInvoker.invoke(ResourceLocatorInvoker.java:104)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:440)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:229)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:135)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.interception.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:358)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:138)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:215)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:245)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:61)
	at org.jboss.resteasy.resteasy-jaxrs@3.13.2.Final//org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:56)
	at javax.servlet.api@2.0.0.Final//javax.servlet.http.HttpServlet.service(HttpServlet.java:590)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:129)
	at org.keycloak.keycloak-wildfly-extensions@12.0.4//org.keycloak.provider.wildfly.WildFlyRequestFilter.lambda$doFilter$0(WildFlyRequestFilter.java:41)
	at org.keycloak.keycloak-services@12.0.4//org.keycloak.services.filters.AbstractRequestFilter.filter(AbstractRequestFilter.java:43)
	at org.keycloak.keycloak-wildfly-extensions@12.0.4//org.keycloak.provider.wildfly.WildFlyRequestFilter.doFilter(WildFlyRequestFilter.java:39)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.security.SecurityContextAssociationHandler.handleRequest(SecurityContextAssociationHandler.java:78)
	at io.undertow.core@2.2.2.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.RedirectDirHandler.handleRequest(RedirectDirHandler.java:68)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57)
	at io.undertow.core@2.2.2.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.core@2.2.2.Final//io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64)
	at io.undertow.core@2.2.2.Final//io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77)
	at io.undertow.core@2.2.2.Final//io.undertow.security.handlers.NotificationReceiverHandler.handleRequest(NotificationReceiverHandler.java:50)
	at io.undertow.core@2.2.2.Final//io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43)
	at io.undertow.core@2.2.2.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.security.jacc.JACCContextIdHandler.handleRequest(JACCContextIdHandler.java:61)
	at io.undertow.core@2.2.2.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.deployment.GlobalRequestControllerHandler.handleRequest(GlobalRequestControllerHandler.java:68)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.SendErrorPageHandler.handleRequest(SendErrorPageHandler.java:52)
	at io.undertow.core@2.2.2.Final//io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:269)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:78)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:133)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:130)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.security.SecurityContextThreadSetupAction.lambda$create$0(SecurityContextThreadSetupAction.java:105)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1530)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1530)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1530)
	at org.wildfly.extension.undertow@21.0.2.Final//org.wildfly.extension.undertow.deployment.UndertowDeploymentInfoService$UndertowThreadSetupAction.lambda$create$0(UndertowDeploymentInfoService.java:1530)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:249)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:78)
	at io.undertow.servlet@2.2.2.Final//io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:99)
	at io.undertow.core@2.2.2.Final//io.undertow.server.Connectors.executeRootHandler(Connectors.java:387)
	at io.undertow.core@2.2.2.Final//io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:841)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1990)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)
	at org.jboss.xnio@3.8.2.Final//org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1280)
	at java.base/java.lang.Thread.run(Thread.java:829)

2021-07-17 13:17:05,754 WARN  [org.keycloak.events] (default task-25) type=REFRESH_TOKEN_ERROR, realmId=master, clientId=security-admin-console, userId=null, ipAddress=127.0.0.1, error=invalid_token, grant_type=refresh_token, client_auth_method=client-secret
2021-07-17 18:10:29,808 WARN  [org.keycloak.events] (default task-26) type=LOGIN_ERROR, realmId=master, clientId=null, userId=null, ipAddress=127.0.0.1, error=expired_code, restart_after_timeout=true, authSessionParentId=4d69333a-3804-4245-8524-307a5e342922, authSessionTabId=kU5WxpD9Xps
2021-07-17 18:10:57,691 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:10:57,693 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (7, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:11:04,861 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:11:04,872 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (8, null, null, null, null, null, null, null, null, null, null, null, pololito, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:12:01,384 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:12:01,384 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (9, null, null, null, null, null, null, null, null, null, null, null, pololito, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:12:12,251 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:12:12,267 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (10, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:14:11,391 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:14:11,391 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (11, null, null, null, null, null, null, null, null, null, null, null, pololukyjtxhfbc, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:14:19,789 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:14:19,789 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (12, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:14:51,647 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:14:51,648 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (13, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:16:43,115 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) SQL Error: 0, SQLState: 23502
2021-07-17 18:16:43,150 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-30) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (14, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:16:58,737 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-26) SQL Error: 0, SQLState: 23502
2021-07-17 18:16:58,738 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-26) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (15, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(new.last_name , new.first_name , new.email , new.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:25:10,950 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-26) SQL Error: 0, SQLState: 23502
2021-07-17 18:25:10,982 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-26) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (16, null, null, null, null, null, null, null, null, null, null, null, polo, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:25:35,963 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-31) SQL Error: 0, SQLState: 23502
2021-07-17 18:25:35,963 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-31) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (17, null, null, null, null, null, null, null, null, null, null, null, polol, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:30:42,735 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:30:42,735 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (18, null, null, null, null, null, null, null, null, null, null, null, lkjhg, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:33:06,127 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:33:06,127 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (19, null, null, null, null, null, null, null, null, null, null, null, lkjhgm, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:33:15,301 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:33:15,301 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (20, null, null, null, null, null, null, null, null, null, null, null, lkjhgm, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:34:56,510 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:34:56,529 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (21, null, null, null, null, null, null, null, null, null, null, null, lkjhgm, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:35:02,181 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:35:02,181 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "adresse" violates not-null constraint
  Detail: Failing row contains (22, null, null, null, null, null, null, null, null, null, null, null, lkjhgmkjh, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username)"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:39:28,817 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:39:28,818 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "email" violates not-null constraint
  Detail: Failing row contains (27, null, null, null, null, null, , null, cef3771e-1eeb-48c5-81df-3547b04cf187, null, null, , nice, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username, adresse, id_keycloak, telephone)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username, '', NEW.id, '')"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:40:07,489 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:40:07,489 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "email" violates not-null constraint
  Detail: Failing row contains (28, null, null, null, null, null, , null, , null, null, , nice, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username, adresse, id_keycloak, telephone)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username, '', '', '')"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:40:11,725 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:40:11,725 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "email" violates not-null constraint
  Detail: Failing row contains (29, null, null, null, null, null, , null, , null, null, , nicem, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username, adresse, id_keycloak, telephone)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username, '', '', '')"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:42:49,118 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) SQL Error: 0, SQLState: 23502
2021-07-17 18:42:49,159 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-35) ERROR: null value in column "email" violates not-null constraint
  Detail: Failing row contains (31, null, null, null, null, null, , null, , null, null, , nicemj, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username, adresse, id_keycloak, telephone)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username, '', '', '')"
PL/pgSQL function function_copy() line 3 at SQL statement
2021-07-17 18:43:38,706 WARN  [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-36) SQL Error: 0, SQLState: 23502
2021-07-17 18:43:38,707 ERROR [org.hibernate.engine.jdbc.spi.SqlExceptionHelper] (default task-36) ERROR: null value in column "email" violates not-null constraint
  Detail: Failing row contains (32, null, null, null, null, null, , null, , null, null, , jlbsa, null).
  Where: SQL statement "INSERT INTO public.utilisateurs(nom, prenom, email, username, adresse, id_keycloak, telephone)
		VALUES(NEW.last_name , NEW.first_name , NEW.email , NEW.username, '', '', '')"
PL/pgSQL function function_copy() line 3 at SQL statement
